harvester/utils/ckan_utils.py

```diff
@@ -283,14 +301,14 @@ def simple_transform(metadata: dict, owner_org: str) -> dict:
     return output
 
 
-def ckanify_dcatus(metadata: dict, owner_org: str) -> dict:
-    ckanified_metadata = simple_transform(metadata, owner_org)
+def ckanify_dcatus(metadata: dict, harvest_source) -> dict:
```

Comment:
provide type hint to be consistent.

Reply:
The HarvestSource class is defined in harvest.py, which imports functions from ckan_util. This results in a circular import error. To resolve this, move the import of the function to the specific location where it is used. Then added the type hint into this function.